_data/current.yml

```diff
@@ -1,3 +1,10 @@
+- name: SEETEST 2018 Conference
```

Comment:
This conference is in the wrong place. It needs to be in the right order and this would place it out of order.

Reply:
resolved it